pct.sh

```diff
@@ -28,7 +28,6 @@ exec java \
 	--working-dir "$(pwd)/target/pct-work" \
 	$PCT_D_ARGS \
 	${PCT_OPTS-} \
-	-Djth.jenkins-war.path="$(pwd)/target/megawar-$LINE.war" \
```

Comment:
This is a far-reaching change that should not be done without discussion.

Reply:
it is broken by design :)

Reply:
https://github.com/jenkinsci/bom/issues/3231 should be used for any discussion about this.

Reply:
split removing the JTH to https://github.com/jenkinsci/bom/pull/3232

Reply:
Merged in

* #3232